packages/phone/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const accountSid = process.env.TWILIO_ACCOUNT_SID;
+const authToken = process.env.TWILIO_AUTH_TOKEN;
+const client = require('twilio')(accountSid, authToken);
+
+const parsePhoneNumber = async ({ countryCode, phoneNumber } = {}) => {
+    if (!countryCode) {
+        throw new Error('A country code is required.');
+    }
+
+    if (!phoneNumber) {
+        throw new Error('A phone number is required.');
+    }
+
+    const phone = await client.lookups.v1
+        .phoneNumbers(phoneNumber)
+        .fetch({ countryCode });
+
+    return phone;
+};
```

Comment:
```suggestion
const parsePhoneNumber = ({ countryCode, phoneNumber } = {}) => {
    if (!countryCode) {
        throw new Error('A country code is required.');
    }

    if (!phoneNumber) {
        throw new Error('A phone number is required.');
    }

    return client.lookups.v1
        .phoneNumbers(phoneNumber)
        .fetch({ countryCode });
};
```
I don't think we need the `async` and `await` parts?

Reply:
@smebberson I think it is better with the `async` because it essentially wraps the errors in a `Promise.reject`. Without it the function could error (not promise.reject), promise reject with the result or promise.resolve if successful.